index.js

```diff
@@ -127,13 +127,14 @@ async function processContent(req, res, wikiContent, extension, archiveFormat) {
     const files = [];
     const images = findImages(wikiContent).map(decodeEntities);
     const urlRE = new RegExp('(' + images.map((i) => i.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&')).join('|') + ')', 'gi');
+
     const content = await replaceAsync(stdout, urlRE, async function(val, grab) {
       if (!images.includes(val)) {
         return val;
       }
 
       try {
-        const filename = `docs/images/${decodeURIComponent(basename(urlParse(grab).pathname)).replace(/\s+/g, '_')}`;
+        const filename = `/docs/images/${decodeURIComponent(basename(urlParse(grab).pathname)).replace(/\s+/g, '_')}`;
```

Comment:
Does the slash make a difference in the zip? In tars you want it all as relative filenames

Reply:
Doesn’t seem to, I’ve been unzipping it fine

Reply:
```suggestion
        const filename = `docs/images/${decodeURIComponent(basename(urlParse(grab).pathname)).replace(/\s+/g, '_')}`;
```

Lets revert it then since its unrelated